exalearn-ui/src/app/components/new-topic/new-topic.component.ts

```diff
@@ -58,11 +64,12 @@ export class NewTopicComponent implements OnInit {
 		void this.apiServise
 			.postRequest(API_PATH.NEW_TOPIC, this.topicForm.value)
 			.then(() => {
+				this.notificationService.successPopUp();
 				this.form.resetForm();
-				this.load = false;
 			})
 			.catch(() => {
-				this.load = false;
-			});
+				this.notificationService.errorPopUp('Sorry. Something went wrong');
+			})
+			.finally(() => (this.load = false));
```

Comment:
Same questions as above

Reply:
We use it while data sending to the server. This property uses for the progress bar. If data is sent ok. We stop the progress bar. If the data sent is not ok, we also stop the progress bar.